test/fixtures/gatherers/saptune-note-list.output

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Just for purity we could remove this first empty line

Reply:
I added it because I noticed the output of `saptune` adds it, but it shouldn't make any difference